src/Components/DurationLineChart.tsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react'
+import { Attempt } from '../types'
+
+import {
+  VictoryAxis,
+  VictoryChart,
+  VictoryLabel,
+  VictoryLine,
+  VictoryScatter,
+  VictoryTheme,
+  VictoryTooltip,
+  VictoryVoronoiContainer
+} from 'victory'
+
+//component for rendering the vertical line on the line chart
+class Cursor extends React.Component<any> {
+  render() {
+    const { x, scale } = this.props
+    const range = scale.y.range()
+    return (
+      <line
+        style={{
+          stroke: 'black',
+          strokeWidth: 0.5
+        }}
+        x1={x}
+        x2={x}
+        y1={Math.max(...range)}
+        y2={Math.min(...range)}
+      />
+    )
+  }
+}
+
+function millisecondsToString(milliseconds: number): string {
```

Comment:
I tend to like putting methods like this inside a separate /utility/ file named PrettyTime. Totally not necessary, but if down the line you want to refactor stuff, here’s a decent place to start.

Reply:
I'll keep that in mind, I debated where to locate the Cursor component, and the three helper functions for a while before just deciding to leave them in the DurationLineChart component file